README.md

```diff
@@ -2,7 +2,13 @@
 
 [![Build Status](https://travis-ci.org/sous-chefs/samba.svg?branch=master)](https://travis-ci.org/sous-chefs/samba) [![Cookbook Version](https://img.shields.io/cookbook/v/samba.svg)](https://supermarket.chef.io/cookbooks/samba)
 
-Installs and configures Samba daemon. Uses Chef Server for data bag to build configuration file shares. Includes a resource for adding / removing Samba users.
+Installs and configures Samba daemon. Uses Chef Server for data bag to build configuration file shares.
+
+# Resources
```

Comment:
This is going to format really oddly if you collapse markdown and seems to duplicate the resources section below